Gems/PhysX/Code/Tests/PhysXColliderComponentModeTests.cpp

```diff
@@ -6,48 +6,71 @@
  *
  */
 
-#include <PhysXColliderComponentModeTests.h>
-
+#include <TestColliderComponent.h>
+
+#include <AzManipulatorTestFramework/IndirectManipulatorViewportInteraction.h>
+#include <AzManipulatorTestFramework/AzManipulatorTestFrameworkTestHelpers.h>
+#include <AzManipulatorTestFramework/AzManipulatorTestFrameworkUtils.h>
+#include <AZTestShared/Math/MathTestHelpers.h>
+#include <AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.h>
+#include <AzToolsFramework/ViewportSelection/EditorInteractionSystemViewportSelectionRequestBus.h>
+#include <AzToolsFramework/ViewportSelection/EditorDefaultSelection.h>
+#include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+#include <AzToolsFramework/Viewport/ViewportSettings.h>
+#include <AzToolsFramework/ViewportUi/ViewportUiManager.h>
+#include <AzToolsFramework/ToolsComponents/EditorNonUniformScaleComponent.h>
 #include <EditorColliderComponent.h>
+#include <EditorRigidBodyComponent.h>
 #include <EditorStaticRigidBodyComponent.h>
-#include <AzToolsFramework/ToolsComponents/EditorNonUniformScaleComponent.h>
 
 namespace UnitTest
 {
-    AZ::Entity* PhysXColliderComponentModeTest::CreateColliderComponent()
+    class PhysXColliderComponentModeTest
+        : public ToolsApplicationFixture<false>
     {
-        AZ::Entity* entity = nullptr;
-        AZ::EntityId entityId = CreateDefaultEditorEntity("ComponentModeEntity", &entity);
+    protected:
+        using EntityPtr = AZ::Entity*;
```

Comment:
nit: As @greerdv mentioned before it's likely fine to remove this alias, it really doesn't add much

Reply:
done

Reply:
Thanks!